tests/unit/search/test_search_queries.py

```diff
@@ -72,6 +73,7 @@ def test_select_source_function_in_author_search():
     assert search_source is None
 
 
+@pytest.mark.xfail(reason="invenio-query-parser updated and not compatible anymore.")
```

Comment:
we could also delete the whole file

Reply:
We are still using some ES queries in workflows as I recall. We should clean this first.